pkg/cmd/server/apis/config/serialization_test.go

```diff
@@ -365,16 +365,16 @@ func fuzzInternalObject(t *testing.T, forVersion schema.GroupVersion, item runti
 			c.FuzzNoCustom(obj)
 			if obj.ResolutionRules == nil {
 				obj.ResolutionRules = []imagepolicyapi.ImageResolutionPolicyRule{
-					{TargetResource: metav1.GroupResource{Resource: "pods"}, LocalNames: true},
-					{TargetResource: metav1.GroupResource{Group: "build.openshift.io", Resource: "builds"}, LocalNames: true},
-					{TargetResource: metav1.GroupResource{Resource: "replicationcontrollers"}, LocalNames: true},
-					{TargetResource: metav1.GroupResource{Group: "extensions", Resource: "replicasets"}, LocalNames: true},
-					{TargetResource: metav1.GroupResource{Group: "batch", Resource: "jobs"}, LocalNames: true},
+					{TargetResource: imagepolicyapi.GroupResource{Resource: "pods"}, LocalNames: true},
```

Comment:
this looks suspicious.  `corev1.GroupResource` maybe?

Reply:
> this looks suspicious. `corev1.GroupResource` maybe?

oh, interesting.  Seems a little odd.  did it have to change?

Reply:
i look at it again.  basically there was a custom implementation of GroupResource that was used in some places, so rather than fight battles about internal/external/etc i just converted everything to use that custom impl.

Reply:
ok i was able to move everything to metav1